Dockerfile

```diff
@@ -38,5 +39,9 @@ RUN mkdir ${KAFKA_HOME} && apt-get update && apt-get install curl -y && apt-get
 COPY --from=kafka_dist /var/tmp/kafka_$scala_version-$kafka_version ${KAFKA_HOME}
 
 RUN chmod a+x ${KAFKA_HOME}/bin/*.sh
+RUN chmod g+rwX ${KAFKA_HOME}
+
+# RUN adduser ${PROJECT_NAME} --disabled-password
```

Comment:
**Note**: to be investigated and either removed or added back in place of the hard coded user.